### PR TITLE
Refactor hustlebook

### LINKED ISSUE #89: Refactor Hustlebook
Refactor occurrences of `AddressBook` to `HustleBook`.

### PR DESCRIPTION
**Pull request by Vishwanth2210**
Currently, the `HustleBook` still has occurrences of the word `AddressBook`. 

Due to the location of these occurrences, this causes confusion amongst developers due to varying names.

Let's,
* Refactors all occurrences of `AddressBook`, `addressBook`, `addressbook` and `address book` in all java classes to `HustleBook`, `hustleBook`, `hustlebook` and `hustle book`
* Update class names from `AddressBook` to `HustleBook`
* Update the Json test files' names from `AddressBook` to `HustleBook`

Closes #89

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -69,25 +69,25 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s hustle book and {@code userPrefs}. <br>
```
Comment (AD-NAP) [inline src/main/java/seedu/address/MainApp.java:72]: I think for consistency, let's use HustleBook instead of hustle book?

Comment (AD-NAP) [reply src/main/java/seedu/address/MainApp.java:72]: ah, i think it's fine. I notice you're just following based on the original text. In that case, it's fine! 😄

Comment (Vishwanth2210) [reply src/main/java/seedu/address/MainApp.java:72]: Alright!

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -167,13 +167,13 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
...
     public void start(Stage primaryStage) {
-        logger.info("Starting AddressBook " + MainApp.VERSION);
+        logger.info("Starting HustleBook " + MainApp.VERSION);
         ui.start(primaryStage);
     }
 
     @Override
     public void stop() {
-        logger.info("============================ [ Stopping Address Book ] =============================");
+        logger.info("============================ [ Stopping Hustle Book ] =============================");
```
[the remark this answers is not in the document]
Comment (Vishwanth2210) [reply src/main/java/seedu/address/MainApp.java:176]: Looks good, I have no issues with changing to this way.

**Review by j4ck990 — APPROVED** [review]

**Pull request merged**